Packs/VirusTotal/Integrations/VirusTotalV3/README.md

```diff
@@ -2158,3 +2158,384 @@ Retrieves result of the last Sigma analysis.
+| VirusTotal.File.attributes.exiftool.FileTypeExtension | String | The file type extension. |
+| VirusTotal.File.attributes.exiftool.Newlines | Number | Number of newlines signs. |
+| VirusTotal.File.attributes.crowdsourced_ids_stats.info | Number | Number of IDS that marked the file as "info". |
+| VirusTotal.File.attributes.crowdsourced_ids_stats.high | Number | Number of IDS that marked the file as "high". |
+| VirusTotal.File.attributes.crowdsourced_ids_stats.medium | Number | Number of IDS that marked the file as "medium". |
+| VirusTotal.File.attributes.crowdsourced_ids_stats.low | Number | Number of IDS that marked the file as "low". |
+| VirusTotal.File.attributes.trid.file_type | String | The TrID file type. |
+| VirusTotal.File.attributes.trid.probability | Number | The TrID probability. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.description | String | Description of the YARA rule. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.source | String | Source of the YARA rule. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.author | String | Author of the YARA rule. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.ruleset_name | String | Rule set name of the YARA rule. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.rule_name | String | Name of the YARA rule. |
+| VirusTotal.File.attributes.crowdsourced_yara_results.ruleset_id | String | ID of the YARA rule. |
+| VirusTotal.File.attributes.names | String | Name of the file. |
+| VirusTotal.File.attributes.type_tag | String | Tag of the type. |
+| VirusTotal.File.attributes.size | Number | Size of the file. |
+| VirusTotal.File.attributes.sha256 | String | SHA-256 hash of the file. |
+| VirusTotal.File.attributes.type_extension | String | Extension of the type. |
+| VirusTotal.File.attributes.tags | String | File tags. |
+| VirusTotal.File.attributes.last_analysis_date | Number | Last analysis date in epoch format. |
+| VirusTotal.File.attributes.ssdeep | String | SSDeep hash of the file. |
+| VirusTotal.File.attributes.md5 | String | MD5 hash of the file. |
+| VirusTotal.File.attributes.sha1 | String | SHA-1 hash of the file. |
+| VirusTotal.File.attributes.magic | String | Identification of file by the magic number. |
+| VirusTotal.File.attributes.meaningful_name | String | Meaningful name of the file. |
+| VirusTotal.File.attributes.threat_severity.threat_severity_level | String | Threat severity level of the file. |
+| VirusTotal.File.attributes.threat_severity.threat_severity_data.popular_threat_category | String | Popular threat category of the file. |
+| VirusTotal.File.attributes.threat_verdict | String | Threat verdict of the file. |
+| VirusTotal.File.type | String | Type of the file. |
+| VirusTotal.File.id | String | ID of the file. |
+| VirusTotal.File.links.self | String | Link to the response. |
+
+#### Command Example
+
+```!vt-privatescanning-file file=f912398cb3542ab704fe917af4a60d4feee21ac577535b10453170f10c6fd6de```
```

Comment:
Please replace any file hash, file path or any other possible sensitive value in this file with a 'dummy value' (`file=exmaple-file-hash` for example).
You can use `demisto-sdk secrets` command to help you locate all possible sensitive values in the file ([more information here](https://xsoar.pan.dev/docs/concepts/demisto-sdk#secrets)).